SEIR/utils.py

```diff
@@ -7,7 +7,7 @@
 import scipy.stats
 import sympy.parsing.sympy_parser
 
-config = confuse.Configuration("COVIDScenarioPipeline")
+config = confuse.Configuration("COVIDScenarioPipeline", read=False)
```

Comment:
What's the read option do?

Reply:
related to the higher version maybe?

Reply:
`read=False` means don't try to read configuration files until you get a `set_config` option. with `read=True`, confuse will look in a few places (e.g., `~/.COVIDScenarioPipeline`) to "smartly" find a config.